Rakefile

```diff
@@ -170,5 +170,21 @@ if ENV["YUPDATE_FORCE"] == "1" || File.exist?("/.packages.initrd") || live_iso?
       FileUtils.mkdir_p(File.join(destdir, "/usr/share"))
       FileUtils.cp_r("playwright/.", File.join(destdir, "/usr/share/agama-playwright"))
     end
+
+    if ENV["YUPDATE_SKIP_PRODUCTS"] != "1"
+      files = Dir.glob("products.d/*.y{a}ml")
+      files.each do |f|
+        # the sources contain several products, update only the existing files
+        oldfile = File.join("/usr/share/agama/", f)
+        if File.exist?(oldfile)
+          target = File.join(destdir, "/usr/share/agama/", f)
+          FileUtils.mkdir_p(File.dirname(target))
+          FileUtils.cp(f, target)
+        else
+          # if there is a new product file it needs to be copied manually
```

Comment:
Without putting too much thinking into it... not sure if this is the behavior I would expect.

Reply:
well, problem is that it is hard to know which product belongs to which iso

Reply:
The problem is that the Git repository contains YAML files for all product. If you are patching the openSUSE Agama installer then the Live medium contains only the openSUSE products.

If yupdate would install all files you would suddenly see the ALP Dolomite product in the list, that would be confusing. So I decided to patch only the existing files and to not blindly copy everything available. IMHO it is better to have a safer default behavior.